x/posts/legacy/v0.13.0/migrate_test.go

```diff
@@ -228,22 +228,19 @@ func TestConvertOptionalData(t *testing.T) {
 		"another":  "data",
 	}
 
-	expOptionalData := []v0130.OptionalDataEntry{
-		{
-			Key:   "optional",
-			Value: "data",
-		},
-		{
-			Key:   "old",
-			Value: "version",
-		},
-		{
-			Key:   "another",
-			Value: "data",
-		},
-	}
-
 	actualOptionalData := v0130.ConvertOptionalData(oldOptionalData)
 
-	require.Equal(t, expOptionalData, actualOptionalData)
+	require.Equal(t, len(oldOptionalData), len(actualOptionalData))
+
+	var found = make([]bool, len(actualOptionalData))
+
+	for key, value := range oldOptionalData {
+		for index, entry := range actualOptionalData {
+			if result := entry.Key == key && entry.Value == value; result {
+				found[index] = true
+			}
+		}
+	}
+
+	require.Equal(t, []bool{true, true, true}, found)
```

Comment:
I think there is an easier way of doing this, without having to use an additional `found` array: 

```suggestion
	for key, value := range oldOptionalData {
	        found := false
		for index, entry := range actualOptionalData {
			found = entry.Key == key && entry.Value == value
			if found {
			    break
			}
		}
		require.True(t, found)
	}
```